docs/docs/38-authoring/04-volumes.md

```diff
@@ -35,6 +35,24 @@ volumes: {
 }
 ```
 
+## Volumes with subpaths
+
+When referencing volumes in containers, subpaths are supported. This feature allows you to have containers share a volume easily. You can utilize volume subpaths by adding a query parameter in the `dir`'s volume url like so.
```

Comment:
Can this start with a definition of what a sub-path is and when you would want to use it? Perhaps provide an example of the volume's filesystem and how it is used with the container here.

Reply:
+1

Reply:
Alright, did an uplift to make this more descriptive. Tell me what you think!